src/components/App.js

```diff
@@ -6,7 +6,7 @@ import ButtonPanel from './ButtonPanel';
 class App extends React.Component {
```

Comment:
Hi, we have a case issue here.
you are importing from `ButtonPanel` with upper case `B` but the file is lower case `b`
- the same issue exits in the `buttonPanel` file.

![Screenshot 2019-11-20 at 4 28 27 PM](https://user-images.githubusercontent.com/10922508/69252179-e3abe900-0bb2-11ea-8084-18b661a72d83.png)